sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -112,6 +112,31 @@ get_wi_data(Group sg, joint_matrix<Group, T, Use, Rows, Cols, Layout> &jm) {
 #endif // defined(__SYCL_DEVICE_ONLY__)
 }
 
+template <typename Group, typename T, use Use, size_t M, size_t N,
```

Comment:
On line 98: an another guard is needed so get_wi_data does not work for JIT path

Reply:
Bing suggested to use "(__attribute__((unavailable))" to get compilation error when things are not available on the JIT path.

Reply:
Updated